libvswitch/src/vswitch.c

```diff
@@ -18,6 +18,10 @@ struct ether_addr bcast_macaddr = { .ether_addr_octet = {
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF
     }
 };
+struct ether_addr ipv6_multicast_macaddr = { .ether_addr_octet = {
+        0x33, 0x33, 0x0, 0x0, 0x0, 0x0
```

Comment:
Can you add a comment about the value of the address, e.g which range is allowed here, which spec defines this...